<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessManagement</class>
 <widget class="QMainWindow" name="ProcessManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProcessManagement</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="Time">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1700</width>
      <height>950</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color:white;

}
QPushButton{
background:#ced1d8;
border-style:outset;
border-radius:20px;
font: 15pt &quot;Mv Boli&quot;;
color:black;
}
QPushButton:pressed{
background-color:rgb(224,0,0);
border-style:inset;
font-style:MingLiU-ExtB;
}
QLabel{
	border:2px solid rgb(0, 0, 0);
font: 15pt &quot;Mv Boli&quot;;
color:black;
}
Line{
color:black;

}
QTableWidget{
color:black;
background-color:white;
background-color:rgba(85,170,255,0);       
font: 15pt &quot;Mv Boli&quot;;
border:2px solid;
gridline-color: black;

}
QProgressBar {
border: 2px solid grey;
border-radius: 5px;
background-color: #FFFFFF;
}

QProgressBar::chunk {
background-color: black;
width: 20px;
}

QProgressBar {
border: 2px solid black;
border-radius: 5px;
text-align: center;
}
QLineEdit{
border:2px solid rgb(0, 0, 0);
background-color:rgb(255, 255, 255);
font: 15pt &quot;Mv Boli&quot;;
border-radius:20px;
color:black;
}
QListWidget{
	border:2px solid black;
color:black;
font: 15pt &quot;Mv Boli&quot;;
}
QListWidget::item{
height:75px;
font: 15pt &quot;Mv Boli&quot;;
border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QPushButton" name="Creat">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>390</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mv Boli</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Creat Process</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Pause">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>60</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mv Boli</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>640</y>
       <width>20</width>
       <height>285</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>630</y>
       <width>1691</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="WaitingLabel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>170</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Mv Boli&quot;;</string>
     </property>
     <property name="text">
      <string>Waiting</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReadyLabel">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>170</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Mv Boli&quot;;</string>
     </property>
     <property name="text">
      <string>Ready</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>20</width>
       <height>621</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>50</y>
       <width>431</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="RunPriority">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="RunTime">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="RunID">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>1190</x>
       <y>10</y>
       <width>20</width>
       <height>621</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ReadyLabel_3">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>170</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Mv Boli&quot;;</string>
     </property>
     <property name="text">
      <string>Hang</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReadyLabel_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Mv Boli&quot;;</string>
     </property>
     <property name="text">
      <string>Running:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReadyLabel_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Mv Boli&quot;;</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReadyLabel_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Mv Boli&quot;;</string>
     </property>
     <property name="text">
      <string>ID     ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReadyLabel_7">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>160</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Mv Boli&quot;;</string>
     </property>
     <property name="text">
      <string>Priority  ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReadyLabel_8">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>230</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Mv Boli&quot;;</string>
     </property>
     <property name="text">
      <string>Time    ：</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="InMemory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>841</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
background-color:white;
background-color:rgba(85,170,255,0);

</string>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>164</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Priority</string>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>State</string>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hang</string>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="Record">
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>20</y>
       <width>471</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
background-color:white;
background-color:rgba(85,170,255,0);
</string>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Priority</string>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RestTime</string>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
    </widget>
    <widget class="QProgressBar" name="RunningBar">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>431</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="NewID">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="NewPriority">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="NewTime">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="Waiting">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>240</y>
       <width>161</width>
       <height>389</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
    <widget class="QListWidget" name="Hang">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>240</y>
       <width>161</width>
       <height>389</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
    <widget class="QListWidget" name="Ready">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>240</y>
       <width>161</width>
       <height>389</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="ByPriority">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>510</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mv Boli</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Priority</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ByRoundRobin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mv Boli</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Round-Robin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="WaitingButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>510</y>
       <width>91</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mv Boli</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Waiting</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReadyLabel_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Mv Boli&quot;;</string>
     </property>
     <property name="text">
      <string>Size    ：</string>
     </property>
    </widget>
    <widget class="QListWidget" name="Storage">
     <property name="geometry">
      <rect>
       <x>868</x>
       <y>648</y>
       <width>804</width>
       <height>274</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NewSize">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>300</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1700</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
